versions/3.0.md

```diff
@@ -989,7 +989,7 @@ When `example` or `examples` are provided in conjunction with the `schema` objec
 
 Field Name | Type | Description
 ---|:---:|---
-<a name="parameterContent"></a>content | Map[`string`, [Media Type Object](#mediaTypeObject)] | A map containing the representations for the parameter. The key is the media type and the value describes it.  The map MUST only contain one entry.
+<a name="parameterContent"></a>content | Map[`string`, [Media Type Object](#mediaTypeObject)] | A map containing the representations for the parameter. The key is the media type and the value describes it.  The key can be a single media type (e.g. `application/json`), a wildcard media type (e.g. `image/*`), or a comma-separated list of the two types. The map MUST only contain one entry.
```

Comment:
The latter. If you have a different wording suggestion, I would be happy to modify.